plat/rockchip/rk3399/include/platform_def.h

```diff
@@ -92,7 +92,7 @@
  * Platform memory map related constants
  ******************************************************************************/
 /* TF txet, ro, rw, Size: 512KB */
-#define TZRAM_BASE		(0x0)
+#define TZRAM_BASE		(0x500000)
```

Comment:
What kind of memory does this are now correspond to? I thought this was SRAM but I can't believe you have over 5MB of SRAM to allow you to move this up so high. Or is this in a 2nd SRAM chip? Or is this DRAM after all? In any case, can the normal world see this memory? If so, this might be another issue you should fix.